src/planet.cpp

```diff
@@ -78,7 +78,7 @@ std::ostream &operator<<(std::ostream &os, const planet &p)
 {
     os << "Planet name: " << p.get_name();
     os << "\nC++ class name: " << boost::core::demangle(value_type_index(p).name()) << '\n';
-    os << fmt::format("mu central body (-1 if not defined): {}\n", p.get_mu_central_body());
+    os << fmt::format("\nmu central body (-1 if not defined): {}\n", p.get_mu_central_body());
```

Comment:
Is there a stray ``\n`` in here?

Reply:
i think I liked it for clarity ... but then I have been going back and forth on this decision

Reply:
yeah nevermind I get it now, you want more vertical spacing.